compilers/concrete-compiler/compiler/tests/python/test_compilation.py

```diff
@@ -439,3 +439,50 @@ def test_crt_decomposition_feedback():
 
     assert isinstance(compilation_feedback, CompilationFeedback)
     assert compilation_feedback.crt_decompositions_of_outputs == [[7, 8, 9, 11, 13]]
+
+
+@pytest.mark.parametrize(
```

Comment:
You should fix the parameters to be sure update on optimizer will not break this tests.

Reply:
I'm not sure we have a way to fix parameters from python, do we? Also, I would see it more as a check to know when the compiler starts using less or more memory